unitary/examples/quantum_chinese_chess/board.py

```diff
@@ -83,47 +97,157 @@ def from_fen(cls, fen: str = _INITIAL_FEN) -> "Board":
         # TODO(): maybe add check to make sure the input fen itself is correct.
         return cls(board, current_player, king_locations)
 
-    def __str__(self):
-        num_rows = 10
-        board_string = ["\n "]
-        # Print the top line of col letters.
-        for col in "abcdefghi":
-            board_string.append(f" {col}")
-        board_string.append("\n")
-        for row in range(num_rows - 1, -1, -1):
-            # Print the row index on the left.
-            board_string.append(f"{row} ")
-            for col in "abcdefghi":
-                piece = self.board[f"{col}{row}"]
+    # TODO(): print players' names in their corresponding side of the board.
+    # To run the game on sublime terminus, set the following sublime_terminus=True
+    # for proer printing.
+    # TODO(): check if there is better way to automatic determine the current terminal
+    # type, e.g. sublime terminus vs glinux / mac terminal.
+    # TODO(): right now all possibilities are printed in black, maybe update to print
+    # in the same color as the corresponding pieces.
+    def to_str(
+        self,
+        probabilities: List[float] = None,
+        print_probabilities=True,
+        peek_result: List[int] = None,
+        sublime_terminus: bool = True,
```

Comment:
I'd consider making this an enum rather than a bool, and add a plain option, such as:

```
class TerminalType(enum.Enum):
    PLAIN_ASCII = 0
    MAC_AND_LINUX = 1
    SUBLIME_TERMINUS = 2

```

Reply:
Thanks Doug. I just added MAC_OR_LINUX and SUBLIME_TERMINUS_OR_COLAB (since it turns out that colab handles full width chars in a similar way as sublime terminus). What do you mean about plain ascii?